Spanish revision of QuickFacts SLA files

BlackLotus64 opened the pull request:
While I was doing a review of some files translated by @lh1008 in the merged PR #30, I found some corrections I would like to present. They're all about Quick Facts files but I've only edited the SLA files. I'm sure this will help with the Spanish grammar.

lh1008 commented on monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.sla:
```diff
@@ -21964,13 +21964,13 @@
-            <ITEXT FONT="Adobe Caslon Pro Semibold" FONTSIZE="12" FEATURES="inherit underline" FCOLOR="quickfacts" CH="construyendo un monedero físico"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" construido sobre las características distintivas de Monero. Ambos monederos físicos se anticipa que estarán operativos para el final del 2018. "/>
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="Monederos físicos (hardware):"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" Los reconocidos monederos de Ledger y Trezor, ambos, están trabajando para agregar a Monero (Ledger tiene soporte desde la versión v0.12.2 para monero-wallet-cli, y v0.12.3 para monero-wallet-gui). La comunidad de Monero está financiando a un equipo de trabajo que está "/>
```
"La comunidad de Monero está financiando a un equipo de trabajo que **esta**" without tilde.

[esta,ésta o está](https://www.diccionariodedudas.com/esta-esta-o-esta/) _**Esta**_ es un demostrativo femenino que puede referirse a lo que está cerca de la persona que habla, a algo que ha sido mencionado o será mencionado a continuación, o a un periodo de tiempo, un momento o un lugar que es próximo. No debe acentuarse en ningún caso.

lh1008 replied:
I will fix the original sentence I suggested, noticed that there are some ambiguitys when reading the text. I'm giving options with arguments. I suggest to do some research and share it inside the comments so we can continue with the workflow. @BlackLotus64 it seems you are straightforward in what you are suggesting on changing. 

"La comunidad de Monero **esta** financiando a un equipo de trabajo que **está**"

- Los determinantes demostrativos nunca han sido tildados: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa. Demonstrative determinants have never been labeled: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa.
- **Está** es la forma verbal en tercera persona del presente del modo indicativo del verbo **estar**. **Está** is the verbal form in the third person of the present in the indicative way of the verb **estar**.

This is an answer given inside a Peruvian university forum from someone asking for help in the issue for the word "esta, ésta, or está".

[Esta, está y ésta](http://udep.edu.pe/castellanoactual/duda-resuelta-esta-esta-y-esta/)

Answer:
>Estimada Rocío:
Una de las novedades de la Ortografía de la lengua española (2010) fue justamente la eliminación de la tilde diacrítica en los demostrativos cuando funcionan como pronombres: Estas se guardan en el cajón de arriba; *Éstas se guardan en el cajón de arriba.
Los determinantes demostrativos nunca han sido tildados: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa.
El demostrativo esta se usa para identificar a algo o a alguien que está cerca del hablante, pero lejos del oyente: Alumnos, observen con atención estas imágenes. Puede encontrar más información sobre este asunto en http://udep.edu.pe/castellanoactual/duda-resuelta-demostrativos/
**Está** es la forma verbal en tercera persona del presente del modo indicativo del verbo **estar**. Y se tilda porque es una palabra aguda terminada en vocal: El café está demasiado caliente.
Saludos cordiales,
Castellano Actual

Translation:
>One of the novelties of the Spelling of the Spanish language (2010) was precisely the elimination of the diacritic tilde in the demonstratives when they function as pronouns: Estas se guardan en el cajón de arriba; *Éstas se guardan en el cajón de arriba.
Demonstrative determinants have never been labeled: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa.
The demonstrative is used to identify something or someone who is close to the speaker, but away from the listener: Alumnos, observen con atención estas imágenes. You can find more information about this matter at http://udep.edu.pe/castellanoactual/duda-resuelta-demostrativos/
**Está** is the verbal form in the third person of the present in the indicative way of the verb **estar**. And it is marked because it is a sharp word ending in a vowel: Coffee is too hot.
Best regards,
Current Spanish

lh1008 replied:
@BlackLotus64 ping

BlackLotus64 replied:
Yes. As you stated, the word _está_ with a tilde is correct because in the next text like it talks about the Monero team developing a new wallet, so it is an action or, an indicative way.

lh1008 replied:
I suggest this:
"La comunidad de Monero **esta** financiando a un equipo de trabajo que está"

Removing the first "está" with tilde and leave the second one as you suggest.

@BlackLotus64

BlackLotus64 replied:
Both should have the tilde since they are indicating actions and not a subject.

lh1008 replied:
ok

lh1008 commented:
Deleted the gender comments. I don't mind "eliminating" some gender words, but we can't deny the female terminology in Spanish. This at some point will be breaking the grammatical structure of the language. What I understood about no using gender was referring to as the example below:

E.g. 
**Look at the script.**
No
**Mira el script.**
Yes
**Mire el script.**

Please correct me if I'm wrong. 

@BlackLotus64  @erciccione

lh1008 commented:
Yeah, @erciccione I do understand and it's ok in some cases, not always, you could read one of the comments above. I explain that some masculine singular words end in a plural "feminine" structure, but they are not giving any preference to gender he/she type.

Singular "sigilo" - Plurar "sigilosas"

That is what I meant in:
>but we can't deny the female terminology in Spanish

Maybe I was not clear. I apologize for that.

lh1008 commented on monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.sla:
```diff
@@ -21680,9 +21680,9 @@
+            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="firmas confidenciales de círculo"/>
             <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="14" CH=" y "/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="stealth addresses"/>
+            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="direcciones de sigilo"/>
```
What I noticed is that it seems we are trying not to use the feminine words, in any case, not even if a masculine word ends in a plural "feminine" way and can be used or "replace" the singular masculine, saying the exact same words. This won't hurt any type of preference in the gender terminology, because is normal to write like that. 

Example (btw, I have never been ok with this translations but we are going to work it out and see how it goes): **Address = Dirección**, and "dirección" is feminine, there is no way on making addresses masculine.
- Singular
"Stealth Address" = "Direccion de sigilo"
- Plural
"Stealth Addresses" = "Direcciones sigilosas (feminine) or Direcciones de sigilo(**"masculine"**)"

*Both can be used, but we can't get into a discussion or "fight" because one expresses itself in plural and the other one in singular plural.

A person can either be "sigilosa" or "sigiloso", and if we say "sigiloso" we will always be marking a "masculine" PERSON gender word. If we say a person is "sigilosa"(una persona es sigilosa) is not referring a feminine gender terminology person (sex, he or she), only as plural of "sigilo". When the word, in this case "address = dirección" (feminine) is next to a singular word "sigilo" or plural "sigilos" (masculine noun), is not right to say "dirección de sigilos", you could either say "direcciones sigilosas" or like the suggested used words "direcciones de sigilo".

I leave this comment because it seems you are avoiding grammatical structures that are agreed in the spanish vocabulary and can also be used without affecting or giving gender preferences.

lh1008 replied:
@BlackLotus64 ping

BlackLotus64 replied:
Waiting for the results of the poll in Telegram :)

BlackLotus64 replied:
The poll ended and the results are: _dirección secreta_. Updating.

lh1008 replied:
Great community support! :)

pikomule commented:
stealth addresses: Dirección indetectable

lh1008 commented:
Stealth addresses: Direcciones ofuscadas u ocultas?

Viking0x commented:
Stealth addresses: Direcciones secretas / Direcciones reservadas

BlackLotus64 commented:
Only waiting for the results of the translation of _stealth addresses_ from a poll in the Spanish chat in Telegram since there's nothing more to change.

And about the gender specific terminology, well, what I catch was that we shouldn't use a preferred gender **articles** when there's a neutral one, and that is all. In Spanish we can't avoid using gender specific *adjectives* since all things and subjects have a gender.

BlackLotus64 commented:
Corrected and updated. I am not able to export the PDF files since I am getting a lot of _text overflow_ errors from these SLA files. I also tried to open to try exporting the original files (in English) but the same error appeared.

Nothing from the file code changed but someone who has already exported PDFs should be able to export them from the SLA files from this PR without any problem. I will still try to solve that problem.

lh1008 commented:
We can merge and I can export the PDF files @BlackLotus64, it's ok. :)

ping to @erciccione this is ready to merge :)

Thank you all for your hard work guys

lh1008 commented:
Thank you guys :).
